packages/svrx/lib/io/index.js

```diff
@@ -46,10 +46,15 @@ class IO {
     });
   }
 
+  // deprecated
```

Comment:
Use `@deprecated` [comment](https://jsdoc.app/tags-deprecated.html) for friendly prompt for editors.

Reply:
done